omnibus/config/software/system-probe.rb

```diff
@@ -32,4 +32,11 @@
   end
 
   copy 'pkg/ebpf/c/COPYING', "#{install_dir}/embedded/share/system-probe/ebpf/"
+
+  if not ENV['NIKOS_PATH'].empty?
+    copy "#{ENV['NIKOS_PATH']}/bin", "#{install_dir}/embedded/nikos/embedded/"
```

Comment:
I think you need some `mkdir` statements before this. I believe this is causing the CI to fail right now.